pkg/settings/setting_test.go

```diff
@@ -54,3 +54,25 @@ func TestSystemDefaultRegistryDefault(t *testing.T) {
 	}
 
 }
+
+// TestSystemFeatureChartRefreshSecondsDefault tests that the default refresh time is either
+// the default value of 21600 seconds or the build time value set through InjectDefaults.
+func TestSystemFeatureChartRefreshSecondsDefault(t *testing.T) {
+	expect := "21600"
+	if InjectDefaults != "" {
+		defaults := map[string]string{}
+		if err := json.Unmarshal([]byte(InjectDefaults), &defaults); err != nil {
+			t.Errorf("Unable to parse InjectDefaults: %v", err)
```

Comment:
Do you want to continue the test execution after this error? If not, you could just use `t.Fatalf` instead and not run any code below that has an assertion too.